server/webapp/WEB-INF/rails/webpack/helpers/spark_routes.ts

```diff
@@ -164,8 +164,8 @@ export class SparkRoutes {
     return `/go/api/internal/config_repos/${id}/trigger_update`;
   }
 
-  static pacListConfigFiles() {
-    return `/go/api/admin/internal/pac/config_files`;
+  static pacListConfigFiles(pluginId?: string) {
+    return pluginId ? `/go/api/admin/internal/pac/config_files?${m.buildQueryString({ pluginId })}` : `/go/api/admin/internal/pac/config_files`;
```

Comment:
This is a POST endpoint so passing the param like this seems off

Reply:
Beyond that, everything looks good to me

Reply:
@ibnc `POST` requests can also accept query params; AFAIK that _shouldn't_ break any conventions?

I could make this part of the path if you think that's better (this would make it required rather than optional, meaning you will only see a single plugin's results) or even the request body (though for the body it's probably more work).

Reply:
@ibnc and maybe it's better as part of the path because, at the moment, we don't need to query more than 1 plugin.